src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -3285,6 +3285,7 @@ public static void Xml_HiddenDerivedFieldTest()
         Assert.Equal(value.value, ((DerivedClass)actual).value);
     }
 
+#if !uapaot
     [Fact]
     public static void Xml_DefaultValueAttributeSetToNaNTest()
```

Comment:
@mconnew  we need to ifdef this test because the test would fail the project's build on UWP. The generated code for this test does not build.